src/components/dropdown/dropdown.tsx

```diff
@@ -0,0 +1,45 @@
+import cn from 'classnames';
+import debounce from 'lodash.debounce';
+import { Person } from '../../types/Person';
+import { DropdownMenu } from '../dropdownMenu/dropdownMenu';
+
+interface Props {
+  people: Person[];
+  value: string;
+  showList: boolean;
+  onIsShowList: (val: boolean) => void;
+  onFilteringPeopleList: (value: string) => void;
+  onChangeSelPerson: (person: Person) => void;
+}
+
+export const Dropdown: React.FC<Props> = ({
+  people,
+  value,
+  showList,
+  onIsShowList,
+  onFilteringPeopleList,
+  onChangeSelPerson,
+}) => {
+  const handleToggleListDelayed = debounce(() => onIsShowList(true), 300);
```

Comment:
Here you can add options for setting debounce delay through props.
```suggestion
  onFilteringPeopleList: (value: string) => void;
  onChangeSelPerson: (person: Person) => void;
  debounceDelay?: number;
}

export const Dropdown: React.FC<Props> = ({
  people,
  value,
  showList,
  onIsShowList,
  onFilteringPeopleList,
  onChangeSelPerson,
  debounceDelay = 300,
}) => {
  const handleToggleListDelayed = debounce(() => onIsShowList(true), debounceDelay);
```